db/migrate/20150317210343_convert_target_minutes_to_hours.rb

```diff
@@ -0,0 +1,13 @@
+class ConvertTargetMinutesToHours < ActiveRecord::Migration
+  class TempTarget < ActiveRecord::Base
+    self.table_name = 'targets'
+  end
```

Comment:
@differentreality yes having an up and down definitely makes more sense. I will push the new changes in the morning. Thanks